### PR TITLE
Always relay to altruist when node relay fails

### PR DESCRIPTION
**Pull request by rem1niscence**
Adds a general try-catch to the node sendRelay process, which allows for the process to continue in case of any failure, also changes the `getCurrentSession` have a maximum number of retries with a different timeout than default.

Note: this PR is currently using the staging version of pocket-js which has not been release yet on npm

### CODE REVIEW

**Review by nymd — DISMISSED** [review]

File: package.json [hunk 1]
```diff
@@ -71,7 +71,7 @@
     "@loopback/rest": "^9.3.1",
     "@loopback/rest-explorer": "3.3.1",
     "@loopback/service-proxy": "^3.2.1",
-    "@pokt-network/pocket-js": "0.6.16-rc",
+    "@pokt-network/pocket-js": "https://github.com/pokt-network/pocket-js.git",
```
Comment (crisog) [inline package.json:74]:
```suggestion
    "@pokt-network/pocket-js": "https://github.com/pokt-network/pocket-js.git#staging",
```

**Review by nymd — APPROVED** [review]

File: src/services/pocket-relayer.ts [hunk 2]
```diff
@@ -479,261 +494,269 @@ export class PocketRelayer {
...
+    const nodesToRemove = await this.redis.smembers(sessionCacheKey)
+
+    if (nodesToRemove.length > 0) {
+      nodes = nodes.filter((n) => !nodesToRemove.includes(n.publicKey))
+    }
 
-      let syncCheckPromise: Promise<Node[]>
-      let syncCheckedNodes: Node[]
+    if (nodes.length === 0) {
+      logger.log('warn', `SESSION: blockchainHash: ${sessionKey} has exhausted all node relays`, {
```
Comment (crisog) [inline src/services/pocket-relayer.ts]: I'd prefer leaving it as before, blockchainHash in the string doesn't provide much value in that log.

`SESSION: ${sessionKey} blah blah`

**Review by crisog — APPROVED** [review]

File: package.json (hunk 1 above)
Comment (nymd) [reply package.json:74]: Staging is default and I won't be messing with that but yes that is a good point.

Comment (rem1niscence) [reply package.json:74]: can be left as is right now yes

**Pull request merged**